exporter/otlpexporter/otlp.go

```diff
@@ -93,7 +100,7 @@ type grpcSender struct {
 	callOptions    []grpc.CallOption
 }
 
-func newGrpcSender(config *Config) (*grpcSender, error) {
+func newGrpcSender(config *Config, _ map[config.ComponentID]component.Extension) (*grpcSender, error) {
```

Comment:
not necessary now.

Reply:
Can we keep it for the time being. The transition will become easy once we modify the ToDIalOption APIs - We don't have to touch a lot test files in the next PR. 

Your call though